src/main/java/seedu/address/model/Model.java

```diff
@@ -145,6 +146,13 @@
      */
     void updateFilteredClientList(Predicate<Client> predicate);
```

Comment:
should this be `updateSortedFilteredClientList`?

Reply:
technically it only updates the filteredclientlist, but the change is "propagated" to the sortedfilteredclientlist, so I'm not sure which would be clearer. I'm thinking since it operates on the filteredclientlist only, the name should be left as is.